cfme/tests/integration/test_cfme_auth.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
+import pytest
+
+from cfme.configure.access_control import Group, User
+from cfme import Credential, login
+from utils.conf import cfme_data
+
+
+def pytest_generate_tests(metafunc):
+    auth_modes = cfme_data['auth_test_data']
+    argvalues = [[authmode] for authmode in auth_modes]
+    idlist = [authmode for authmode in auth_modes]
```

Comment:
I guess you can just pass auth_modes in place of idlist, this will just unnecessarily go through the auth_modes and create a new list.